sql/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java

```diff
@@ -253,4 +256,101 @@ private void assertRow(String expected) {
             throw new AssertionError(msg);
         }
     }
+
+    @Test
+    public void testShowSchemas() throws Exception {
+        execute("create table my_s1.my_table (id long) clustered into 1 shards with (number_of_replicas='0')");
+        execute("create table my_s2.my_table (id long) clustered into 1 shards with (number_of_replicas='0')");
+        execute("show schemas like 'my_%'");
+        assertThat(TestingHelpers.printedTable(response.rows()), is("my_s1\nmy_s2\n"));
```

Comment:
tests are more readable if you break the actual string at the ``\n`` so every row is a row in the test source